service/src/main/java/com/ge/predix/acs/attribute/connectors/AttributeReader.java

```diff
@@ -5,5 +5,5 @@
 import com.ge.predix.acs.model.Attribute;
 
 public interface AttributeReader {
-    Set<Attribute> getAttributes(String identifier);
+    Set<Attribute> getAttributes(final String identifier);
```

Comment:
Concrete classes implementing interface methods should declare the parameters as `final` so this is a redundant use of it. This is also supposed to be enforced by our CheckStyle rules via the `RedundantModifier` module but it's not due to a bug in the version of CheckStyle we're using. I've updated to the latest version of CheckStyle on the `develop` branch so please rebase this branch onto `develop` and rebuild; once you do so you'll see this issue being flagged properly.